lib/mistral_client/mixins/definable.rb

```diff
@@ -7,7 +7,7 @@ def parse_definition(definition)
 
         definition = File.read(definition) if File.exist?(definition)
         # Called outside the if/else to validate the YAML.
-        parsed = YAML.safe_load(definition, [], [], true)
+        parsed = YAML.safe_load(definition, permitted_classes: [], permitted_symbols: [], aliases: true)
```

Comment:
love this. so much more readable

Reply:
yeah! Pushed another change actually, because we don't need to set the classes/symbols. These arrays are  additive to a default list, so passing in empty ones has no effect.

Reply:
nice!